### DISCUSSION TITLE
evalexpr changed to AGPL License with evalexpr version 12

### DISCUSSION THREAD

**Discussion by davidrheimann** [sagiegurari/duckscript#478] [General]
I noticed that evalexpr is now AGPL licensed (as of version 12).  Did we mean to bump up to that version for duckscript on 10/17 in cargo.toml?

**Comment by sagiegurari** [comment]
i let depend bot to upgrade for me. is there anything missing?

**Comment by ant32** [comment]
@sagiegurari The transition to the AGPL implies that individuals must share the source code of their projects, even if the code operates behind a network. As I understand it, this essentially restricts Duckscript to source-available projects only. It might be wise for Duckscript to include a disclaimer clarifying that users must comply with the terms of the AGPL license.

**Reply by sagiegurari** [reply]
duckscript is apache license which does not require users to share source code

**Reply by sagiegurari** [reply]
ok now i read the start of the thread. so worth reverting that change probably. i want to keep this oroject apache license and not gpl nonsense....
